Traceback (most recent call last):
  File "skema/program_analysis/model_coverage_report/model_coverage_report.py", line 78, in generate_data_product
    raise Exception("Data product is empty")
Exception: Data product is empty
